test/new-e2e/tests/process/windows_test.go

```diff
@@ -246,6 +246,8 @@ func (s *windowsTestSuite) TestUnprotectedProcessCheckIO() {
 
 func (s *windowsTestSuite) TestManualProcessCheck() {
 	flake.Mark(s.T())
+	// test can be flaky due to missing CPU stats when cpu usage is extremely low (json output omits 0 values), so we want to re-run a full scan to ensure we have CPU stats
+	s.Env().RemoteHost.MustExecute("Start-MpScan -ScanType FullScan -AsJob")
```

Comment:
If we need to do this for more tests, we might consider putting this in the test-level setup in the future. Currently I see this is in the suite-level setup.